src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImpl.java

```diff
@@ -26,7 +26,7 @@
     @Autowired
     AuthTokenGenerator authTokenGenerator;
 
-    static final String successMessage = "Jurisdiction create user profile success!!";
+    static final String SUCCESS_MESSAGE = "Jurisdiction create user profile success!!";
```

Comment:
remove this string as-well if not used other places

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/1ca60a5369b53bae237ab6c14a9b9df1c4e83cf7